grails-app/domain/org/pih/warehouse/product/Product.groovy

```diff
@@ -596,15 +596,19 @@ class Product implements Comparable, Serializable {
         }
     }
 
-    def validateRequiredFields() {
+    Boolean validateRequiredFields() {
+        return this.validateRequiredFields([])
```

Comment:
`this` is redundant

Reply:
btw I think this method is unnecessary, if you just specify a default argument for that in the line `603` as `[]`, so:
```groovy
 Boolean validateRequiredFields(List<ProductField> ignoreProductFields = [])
```